tuf_conformance/test_file_download.py

```diff
@@ -160,10 +164,9 @@ def test_multiple_changes_to_target(
             # Modify artifact content without updating the hashes/length in metadata.
             malicious_file_contents = f"malicious contents {i}".encode()
             repo.artifacts[target_path].data = malicious_file_contents
-            repo.targets.version += 1
 
             # Bump repo snapshot
```

Comment:
Could specify that we also bump targets and timestamp.

Reply:
https://github.com/theupdateframework/tuf-conformance/pull/188/commits/abacf0b7b4bd7d2ba7f412802f3d3f10a2718f74